package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "codejam",
+  "version": "1.0.0",
+  "eslintConfig": {
+    "env": {
+      "browser": true,
+      "node": true,
+      "mocha": true
+    }
+  },
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/OlgaLazarevich/rsschool-codejam.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/OlgaLazarevich/rsschool-codejam/issues"
+  },
+  "homepage": "https://github.com/OlgaLazarevich/rsschool-codejam#readme",
+  "dependencies": {},
+  "devDependencies": {
+    "eslint": "^5.8.0",
+    "eslint-config-airbnb": "^17.1.0",
+    "eslint-config-airbnb-base": "file:node_modules/eslint-config-airbnb-base",
```

Comment:
That's a bit odd...
`"eslint-config-airbnb": "^17.1.0",`
`"eslint-config-airbnb-base": "file:node_modules/eslint-config-airbnb-base",`
Instead of this you could simply install just `eslint-config-airbnb-base` with the latest version.